docs/workflow/building/libraries/freebsd-instructions.md

```diff
@@ -7,7 +7,7 @@ Tested on plain FreeBSD 11.3 Azure image
 ## Prerequisites
 This needs to be done once on fresh system.
 
-```sudo pkg install cmake git icu libunwind bash python2 krb5 lttng-ust llvm60 libgit2```
+```sudo pkg install cmake git icu libunwind bash python2 krb5 lttng-ust llvm90 libgit2 libinotify```
```

Comment:
nits:

* `openssl` i missing
* `libunwind` is not required for quite some time
* successfully complied coreclr, libraries and installers without `libgit2`, so it can be removed (was this dependency ever required?)
* `python2` can be simplified to `python` (as scripts are compatible with both v2 and v3)

Reply:
openssl should be part of the base unless you want 1.1, right? 
For libunwind FreeBSD and OS X uses system one instead of internal copy. (can be possibly changed) 

I think documentation needs general update but I'm not ready for it yet. I think this is clone from general source-build instructions and that did more than just runtime.

Reply:
Base system does not have openssl package installed. Even the free public FreeBSD VM from Google Cloud Platform, this package is not present. Like almost everything on FreeBSD, we always need to install openssl.

Reply:
```
[furt@freebsd-11 /mnt/wfurt-runtime]$ /usr/bin/openssl version
OpenSSL 1.0.2s-freebsd  28 May 2019
[furt@freebsd-11 /mnt/wfurt-runtime]$ /usr/local/bin/openssl version
OpenSSL 1.1.1d  10 Sep 2019
```
I added openssl and I'll make more updates to docs one everything is stable again.

Reply:
Packages that are preinstalled in minumum base 12.1 (+few to work on GCP): https://api.cirrus-ci.com/v1/task/5847589429968896/logs/pkg_info.log